src/integration-builder/initialization.js

```diff
@@ -62,14 +69,15 @@ var initialization = {
                         key = key.replace(/\D/g, '');
                     }
 
-                    if (this.getConsentGroupIds().indexOf(key) > -1) {
-                        boolean = true;
+                    boolean = (this.getConsentGroupIds().indexOf(key) > -1);
```

Comment:
i think we should change `boolean` to `consentBoolean`.  i know you're just updating it based on what was already here, but I think it's more descriptive.